run.py

```diff
@@ -32,8 +33,17 @@
 PathLike = Union[str, pathlib.Path]
 
 
-def _get_script_modules(*, config, root_dir, subject, session, task, run,
-                        interactive, n_jobs):
+def _get_script_modules(
+    *,
+    config: PathLike,
+    root_dir: Optional[PathLike] = None,
+    subject: Optional[str] = None,
+    session: Optional[str] = None,
+    task: Optional[str] = None,
+    run: Optional[str] = None,
+    interactive: Optional[str] = None,
+    n_jobs: Optional[str] = None
+) -> Dict[str, Tuple[ModuleType]]:
```

Comment:
What kind of declaration is this? @hoechenberger

Reply:
These are called "type hints" and I think were introduced with Python 3.6 initially.

You need Python 3.7 or newer for these here to work.